integration_tests/features/steps/fal_steps.py

```diff
@@ -252,7 +257,8 @@ def _verify_node_order(context):
     assert_precedes = partial(_assert_precedes, ordered_nodes)
     assert_succeeds = partial(_assert_succeeds, ordered_nodes)
     for node in ordered_nodes:
-        if _is_script(node):
+        # Skip all the nodes that are not part of the graph.
```

Comment:
this happens?

Reply:
`ordered_nodes` contain both 'real scripts' and `pre-hooks`/`post-hooks` which are not part of the graph.